packages/contracts/contracts/minter-suite/Minters/MinterSEAV1.sol

```diff
@@ -0,0 +1,1215 @@
+    {
+        return _maxInvocationsProjectConfigMapping[_coreContract][_projectId];
+    }
+
+    /**
+     * @notice projectId => has project reached its maximum number of
+     * invocations? Note that this returns a local cache of the core contract's
+     * state, and may be out of sync with the core contract. This is
+     * intentional, as it only enables gas optimization of mints after a
+     * project's maximum invocations has been reached. A false negative will
+     * only result in a gas cost increase, since the core contract will still
+     * enforce a maxInvocation check during minting. A false positive is not
+     * possible because the V3 core contract only allows maximum invocations
+     * to be reduced, not increased. Based on this rationale, we intentionally
+     * do not do input validation in this method as to whether or not the input
+     * @param `_projectId` is an existing project ID.
+     * @param `_coreContract` is an existing core contract address.
+     */
+    function projectMaxHasBeenInvoked(
+        uint256 _projectId,
+        address _coreContract
+    ) external view returns (bool) {
+        return
+            _maxInvocationsProjectConfigMapping[_coreContract][_projectId]
+                .maxHasBeenInvoked;
+    }
+
+    /**
+     * @notice projectId => project's maximum number of invocations.
+     * Optionally synced with core contract value, for gas optimization.
+     * Note that this returns a local cache of the core contract's
+     * state, and may be out of sync with the core contract. This is
+     * intentional, as it only enables gas optimization of mints after a
+     * project's maximum invocations has been reached.
+     * @dev A number greater than the core contract's project max invocations
+     * will only result in a gas cost increase, since the core contract will
+     * still enforce a maxInvocation check during minting. A number less than
+     * the core contract's project max invocations is only possible when the
+     * project's max invocations have not been synced on this minter, since the
+     * V3 core contract only allows maximum invocations to be reduced, not
+     * increased. When this happens, the minter will enable minting, allowing
+     * the core contract to enforce the max invocations check. Based on this
+     * rationale, we intentionally do not do input validation in this method as
+     * to whether or not the input `_projectId` is an existing project ID.
+     * @param `_projectId` is an existing project ID.
+     * @param `_coreContract` is an existing core contract address.
+     */
+    function projectMaxInvocations(
+        uint256 _projectId,
+        address _coreContract
+    ) external view returns (uint256) {
+        return
+            uint256(
+                _maxInvocationsProjectConfigMapping[_coreContract][_projectId]
+                    .maxInvocations
+            );
+    }
```

Comment:
Inclined a bit to make this something that just gets exposed by `MaxInvocationsLib` (and takes in a `_maxInvocationsProjectConfigMapping`), but maybe not worth – just know that this definitely exists across lots (all) of minters.

Reply:
The same thought applies to `projectMaxHasBeenInvoked` (which is actually a bit of a misnomer and maybe `projectMinterMaxHasBeenInvoked` would be better).
Personally, I like this suggestion (as ideally the minter does not need to be aware of struct variable names), and have added it as a follow-on PR task in Asana here: https://app.asana.com/0/0/1205103275547354/f
(since this has an asana task, going to resolve in this PR 👍 )

Reply:
Fantastic – sounds great to me ser 🙏